src/main/java/frc/robot/commands/FullLiftCommand.java

```diff
@@ -0,0 +1,37 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.Lift;
+
+public class FullLiftCommand extends Command {
+    private Lift lift = Lift.getInstance();
+
+    public FullLiftCommand() {
+        addRequirements(lift);
+    }
+
+    @Override
+    public void initialize() {
+        lift.disableLeftRatchet();
+        lift.disableRightRatchet();
+    }
+
+    @Override
+    public void execute() {
+        lift.runLeftUp();
+        lift.runRightUp();
+    }
+
+    @Override
+    public boolean isFinished() {
+        return !lift.getLiftEncoder();
```

Comment:
This command is only different from the existing ones in the finish condition and I would say the current up command could use the same finish condition so I would just make changes to the existing command